apache/README.md

```diff
@@ -10,6 +10,7 @@ The Apache check tracks requests per second, bytes served, number of worker thre
 The Apache check is packaged with the Agent. To start gathering your Apache metrics and logs, you need to:
 
 1. [Install the Agent](https://app.datadoghq.com/account/settings#agent) on your Apache servers.
+  If you need the newest version of the ActiveMQ XML check, install the `dd-check-apache` package; this package's check overrides the one packaged with the Agent. See the [integrations-core repository README.md for more details](https://github.com/DataDog/integrations-core#installing-the-integrations).
```

Comment:
This should say `apache` instead of ActiveMQ XML

Reply:
👍

Reply:
This is corrected with the last commit, Idk why it's not outdated